### DISCUSSION TITLE
Currency Conversion for volatile Currencies

### DISCUSSION THREAD

**Discussion by SaiKiru** [cloudbet/docs#82] [Q&A]
How do you manage bets for volatile currency such as BTC? This is especially important for errors that the stake is too high or too low. I read in a discussion that the returned response is still in the provided currency (and not EUR), so knowing the conversion used for a particular time in betting could help us adjust the stake better.

**Comment by kgravenreuth** [comment]
Hi @SaiKiru 
Thanks for your feedback. We will consider adding a currency request param for the Cloudbet Feed API to make this process smoother. I'll update this thread if or when we have an update here from the dev team. 

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
